src/components/Landing/Landing.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  Box,
+  Button,
+  Typography,
+  Card,
+  CardMedia,
+  CardContent,
+  Grid,
+} from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import LoginIcon from "@mui/icons-material/Login";
+import StarHalfIcon from "@mui/icons-material/StarHalf";
+import RestaurantIcon from "@mui/icons-material/Restaurant";
+import GetAppIcon from "@mui/icons-material/GetApp";
+import ShareIcon from "@mui/icons-material/Share";
```

Comment:
I'd rather have a whole set of named imports for these icons in a single import, since they are all coming from the same package.